tools/install_demo_configuration.sh

```diff
@@ -113,17 +113,17 @@ SUDO_CMD=""
 ES_INSTALL_TYPE=".tar.gz"
 
 #Check if its a rpm/deb install
-if [ -f /usr/share/elasticsearch/bin/elasticsearch ]; then
-    ES_CONF_FILE="/usr/share/elasticsearch/config/elasticsearch.yml"
+if [ -f $BASE_DIR/bin/elasticsearch ]; then
+    ES_CONF_FILE="$BASE_DIR/config/elasticsearch.yml"
 
     if [ ! -f "$ES_CONF_FILE" ]; then
         ES_CONF_FILE="/etc/elasticsearch/elasticsearch.yml"
     fi
 
-    ES_BIN_DIR="/usr/share/elasticsearch/bin"
-    ES_PLUGINS_DIR="/usr/share/elasticsearch/plugins"
-    ES_MODULES_DIR="/usr/share/elasticsearch/modules"
-    ES_LIB_PATH="/usr/share/elasticsearch/lib"
+    ES_BIN_DIR="$BASE_DIR/bin"
```

Comment:
@epol Thank you for your contribution.
These values are already set above so I was wondering if it is possible to delete these lines instead. Thanks!

Reply:
Thank you for your review. I'm going to write a better edit since this one may have been breaking the rpm/deb logic.
I'll update this pull request as soon as i'm done.